### PR TITLE
🎨 Rewrite referrer checking logic in resize image api

### PR DESCRIPTION
**Pull request by williamchong007**

### CODE REVIEW

File: packages/resizeImageHttp/index.js [hunk 1]
```diff
@@ -15,14 +16,20 @@ const app = express();
...
-  const { origin, referer } = req.headers;
-  if (!IS_TESTNET && origin && !origin.includes(ORIGIN_DOMAIN)) {
-    res.sendStatus(403);
-    return;
-  }
-  if (!IS_TESTNET && referer && !referer.includes(ORIGIN_DOMAIN)) {
-    res.sendStatus(403);
-    return;
+  const { referer } = req.headers;
+  if (referer) {
```
Comment (nnkken) [inline packages/resizeImageHttp/index.js]: Why drops the `origin` header checking?

Comment (williamchong007) [reply packages/resizeImageHttp/index.js]: at least chrome does not send it when fetching img src, only referrer is sent
.... maybe useful for a canvas loaded img though, not sure

Comment (nnkken) [reply packages/resizeImageHttp/index.js]: But then keeping it should not harm, and some browser does not send `referer` header, so keeping the `origin` should in theory catch more cases?

Comment (williamchong007) [reply packages/resizeImageHttp/index.js]: i doubt if they dont send `referer` would they send `origin`....
still we can keep it

**Pull request merged**